column_buffer.go

```diff
@@ -684,7 +684,7 @@ func newBooleanColumnBuffer(typ Type, columnIndex int16, bufferSize int) *boolea
 	return &booleanColumnBuffer{
 		booleanPage: booleanPage{
 			typ:         typ,
-			values:      make([]bool, 0, bufferSize),
+			values:      make([]byte, 0, bufferSize),
```

Comment:
The contract is implicitly changing here, since callers will be able to store 8x as many values in `bufferSize` as they could previously, I guess it's ok though.

Reply:
I'm also nervous about someone attempting to read/write `values` directly, in Python or whatever I'd put `_values` to make that clear, but probably unnecessary here.

Reply:
The semantics don't change since the buffer size is intended to be expressed in bytes. It's true that the buffer can hold 8x more values for the same space then.

Reply:
I'm going to rename from `values` to `bits` to be more explicit that we are bit-packing the booleans 👍